### ISSUE TITLE
Custom linters are not resolved when current directory is a child directory

### ISSUE THREAD

**Issue by WattyRev** [lesshint/lesshint#372]
**Which version of `lesshint` are you using?**
3.1.1

**How are you running `lesshint`? CLI, Node.js API, Grunt/Gulp plugin?**
`lesshint *` from inside a child folder or via Atom lesshint plugin

**What's your `.lesshintrc` configuration?**

```js
{
    "fileExtensions": [".less", ".css"],

    "excludedFiles": [],

    // Custom linters
    "linters": [
        "lesshint-linters/color_variables.js" // NOTE, same result with ./lesshint-... or /leshint-...
    ],

    "colorVariables": {
        "enabled": true,
        "severity": "error"
    },

    // Learn about linters here: https://github.com/lesshint/lesshint/blob/master/lib/linters/README.md

    //... normal linter configs would be below here
}

```

**What did you expect to happen?**
I expected that my custom linter would get resolved when using the Atom lesshint plugin and when running `lesshint .` from a child folder.

**What's the actual result?**
When running `lesshint .`, an error is logged to the console:
```
An unknown error occurred when checking "/BitTitan/FrontEnd/bittitan-css/lib/bittitan-ui-kit/2.0/framework/framework.less", please file an issue with this info:
Error: Cannot find module 'lesshint-linters/color_variables.js'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.require.resolve (internal/module.js:16:19)
    at requireConfigLinter (/Users/swatson/.nvm/versions/node/v4.6.1/lib/node_modules/lesshint/lib/linter.js:234:32)
    at Array.map (native)
    at mergeLintersFromConfig (/Users/swatson/.nvm/versions/node/v4.6.1/lib/node_modules/lesshint/lib/linter.js:222:49)
    at runChecks (/Users/swatson/.nvm/versions/node/v4.6.1/lib/node_modules/lesshint/lib/linter.js:188:21)
    at Object.exports.lint (/Users/swatson/.nvm/versions/node/v4.6.1/lib/node_modules/lesshint/lib/linter.js:255:23)
    at Lesshint.checkString (/Users/swatson/.nvm/versions/node/v4.6.1/lib/node_modules/lesshint/lib/lesshint.js:107:29)
    at /Users/swatson/.nvm/versions/node/v4.6.1/lib/node_modules/lesshint/lib/lesshint.js:74:34
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:380:3)
```

When opening a file in Atom, lesshint provides the same error mentioned above in the form of a popup.

**Would you be interested in submitting a PR for this issue?**
I can add my custom rule to the list of rules, and have already done that in a fork, which I think circumvents the problem, but it doesn't solve the issue of being unable to effectively use custom linters, and I'm not sure what the solution to that is.

**Comment by WattyRev** [comment]
I played around with this some more, and I think I found a way to make this work how I expected it to. Pull request here: https://github.com/lesshint/lesshint/pull/373

**Comment by jwilsson** [comment]
Closed in #375.
